README.md

```diff
@@ -62,25 +62,29 @@ yarn link @code-dot-org/ml-playground
 
 This will set up a symlink in main repo's `apps/node_modules/` to point at your local changes. 
 
-Run 
 
-```
-yarn run build
-```
-
-in this repo, and then the main repo's `apps` build should pick the changes up next time it builds.
+### Building changes for a local Code Studio
+In main repo:
 
-If you are running `yarn start` for continuous builds in the main repo, it will pick up the changes once the build in this repo has completed.
+``` 
+bin/dashboard-server 
+``` 
 
-### Building changes for a local Code Studio
+In main repo's `apps/` directory, run 
+``` 
+yarn start 
+```
 
-Borrowing one step from above:
+If you see an error, run ``` yarn ``` before running ``` yarn start ``` again.
```

Comment:
Inline code just needs to be wrapped by single `s.

Reply:
TIL. Thanks!